### ISSUE TITLE
Unclear error message: "The node version is inconsistent with Neuron(v 0.110.0), please use after confirmation"

### ISSUE THREAD

**Issue by jordanmack** [nervosnetwork/neuron#2780]
**Describe the bug** (Please enter details, the more detail the better)
I use Neuron with a CKB node on a different server in my local network. The CKB node is of a different version, which causes Neuron to give the error message: "The node version is inconsistent with Neuron(v 0.110.0), please use after confirmation".

This message needs to be rewritten to be more clear and fix the spacing and missing punctuation. 

**Debug info** (Please upload the debug info from `Help` - `Export Debug Information`)
[neuron_debug_1689832605961.zip](https://github.com/nervosnetwork/neuron/files/12102464/neuron_debug_1689832605961.zip)

**Screenshots** (If applicable, add screenshots to help explain your problem.)
![20230719225249](https://github.com/nervosnetwork/neuron/assets/37931/fde8262b-0393-4748-ab50-04405b163bf7)

**Comment by Keith-CY** [comment]
How about "The CKB Node version is inconsistent with Neuron(v0.110.0). Please ensure the compatibility before proceeding."

**Comment by jordanmack** [comment]
> The CKB Node version is inconsistent with Neuron(v0.110.0). Please ensure the compatibility before proceeding.

How does a user verify compatibility?

**Comment by Keith-CY** [comment]
> > The CKB Node version is inconsistent with Neuron(v0.110.0). Please ensure the compatibility before proceeding.
> 
> How does a user verify compatibility?

We're building a troubleshooting page for Neuron, and a link in the dialog could point to a compatibility table page as https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API#browser_compatibility

Besides, the compatibility table for specific version could be delivered within the release notes

Ref: 
- Troubleshooting page: https://github.com/Magickbase/neuron-public-issues/issues/99

**Comment by jordanmack** [comment]
@Keith-CY I suggest the following:

```
The current version of Neuron (v0.110.0) does not match the CKB Node version that is configured for use. This could potentially result in compatibility problems and incorrect operation. Before proceeding, please ensure compatibility using the instructions at the URL below.

https://url-to-the-compatibility-table
```

Improving the functionality one step further would be to include the table in Neuron so it can determine compatibility by itself and display this dialog only when an incompatibility exists.

**Comment by Keith-CY** [comment]
> @Keith-CY I suggest the following:
> 
> ```
> The current version of Neuron (v0.110.0) does not match the CKB Node version that is configured for use. This could potentially result in compatibility problems and incorrect operation. Before proceeding, please ensure compatibility using the instructions at the URL below.
> 
> https://url-to-the-compatibility-table
> ```
> 
> Improving the functionality one step further would be to include the table in Neuron so it can determine compatibility by itself and display this dialog only when an incompatibility exists.

The copy is clear and cc @Danie0918 @Sven-TBD

---

Including the compatibility table in neuron has a drawback that the compatibility table may not be updated timely and confuses users. Say Neuron@v0.110.0 has been released with CKB@v0.110.0, and a new release of CKB@v0.111.0 comes, CKB@v0.111.0 may work with Neuron@v0.110.0 but it cannot be recognized by Neuron@v0.110.0.

**Comment by jordanmack** [comment]
> Including the compatibility table in neuron has a drawback that the compatibility table may not be updated timely and confuses users. Say Neuron@v0.110.0 has been released with CKB@v0.110.0, and a new release of CKB@v0.111.0 comes, CKB@v0.111.0 may work with Neuron@v0.110.0 but it cannot be recognized by Neuron@v0.110.0 with the embedded table.

If the CKB node version is newer than Neuron, you can still display a warning message. In this way, no message is displayed unless there is a known problem or Neuron is older than the CKB node.

**Comment by Keith-CY** [comment]
> If the CKB node version is newer than Neuron, you can still display a warning message. In this way, no message is displayed unless there is a known problem or Neuron is older than the CKB node.

I see, we will arrange this feature in next versions.

**Comment by jordanmack** [comment]
> I see, we will arrange this feature in next versions.

@Keith-CY Here is a slightly different message that I would suggest for use in the meantime since a compatibility table is not yet ready.

`The current version of Neuron (v0.110.0) does not match the CKB Node version that is configured for use. This could potentially result in compatibility problems and incorrect operation. Please consider updating Neuron and the CKB Node to matching versions to guarantee proper compatibility.`

**Comment by joshyates1980** [comment]
This was a helpful post. I recently began to run ckb v0.111.0 on older laptop, next installed Neuron. I closed my Neuron v0.110.1 due to this message box because I assumed my full node was going to run simultaneously with internal Neuron version. See my post here: [2848](https://github.com/nervosnetwork/neuron/discussions/2848)

**Comment by Keith-CY** [comment]
A compatibility table has been added to this repo at https://github.com/nervosnetwork/neuron/blob/develop/compatible.json and it will be rendered as a user-friendly table on neuron project page https://neuron.magickbase.com/
